### ISSUE TITLE
Is this same as your online OCR?

### ISSUE THREAD

**Issue by 9tontruck** [A9T9/Free-Ocr-Windows-Desktop#2]
Hi,

I couldn't have it working in my computer yet but just looked through your code.
It seems that it just uses Tesseract-OCR's basic function to detect texts. Which is:

> using (var engine = new TesseractEngine(_pathToTestData, Language, EngineMode.Default))
>                 {
>                     using (var img = Pix.LoadFromFile(pathToImage))
>                     {
>                         using (var page = engine.Process(img))
>                         {
>                             var resultText = page.GetText();
>                             if (!String.IsNullOrEmpty(resultText))
>                                 return resultText;
>                         }
>                     }
>                 }

I know about this code and it wasn't very great. However your online OCR (https://ocr.space/) works really great. How come they are different? Are they really using same code?

I guess they are using different tessdata. Would you be able to share your tessdata too?

**Comment by A9T9** [comment]
You are right, both projects use a different OCR engine. This blog posts explains it: http://blog.a9t9.com/2015/09/ocr-api.html
